mono/metadata/object.c

```diff
@@ -5957,6 +5957,31 @@ ves_icall_array_new_specific (MonoVTable *vtable, uintptr_t n)
 }
 
 /**
+ * mono_string_empty_wrapper:
+ *
+ * Returns: The same empty string instance as the managed string.Empty
+ */
+MonoString*
+mono_string_empty_wrapper ()
+{
```

Comment:
I was following the same pattern as mono_string_new & mono_string_new_wrapper.  Would you still prefer I remove mono_string_empty_wrapper?